tests/test_data/rhsvc/get_fulfilment_single_sms.json

```diff
@@ -7,7 +7,7 @@
        "fulfilmentCode": "UACIT1",
        "handler": "NOTIFY",
        "initialContactCode": "",
-       "language": "eng",
+       "language": "E",
```

Comment:
A lot of the attributes remain the same allowing this to work but not quite accurate for a RH service mocked get fulfilment return JSON?